packages/xo-web/src/xo-app/backup/health/index.js

```diff
@@ -101,6 +101,7 @@ export default class Health extends Component {
                   data-vms={this.props.vms}
                   emptyMessage={_('noSnapshots')}
                   shortcutsTarget='.lone-snapshots'
+                  stateUrlParam='s_vms'
```

Comment:
```suggestion
                  stateUrlParam='s_snapshots'
```

Reply:
VDI snapshots or VM snapshots?

Reply:
I tried to keep it short but yes, `s_vm_snapshots` if fine too.